api-common-java/pom.xml

```diff
@@ -131,4 +131,19 @@
             </plugin>
         </plugins>
     </build>
+
+    <!-- Skip api-common when analyzing showcase test coverage on SonarCloud   -->
```

Comment:
Why do we skip `api-common`? It's also a handwritten runtime dependency of client libraries, same as gax.

Reply:
Currently the test coverage of showcase is restricted to the showcase and gax modules because of how heavily dependent GAPIC clients are on gax (which contains a lot of the transport logic). In terms of development, a feature or a bug fix oftentimes has a corresponding change in gax.  

However,  an argument can be made in support of api-common being included. The coverage report for showcase is to provide insights into how much of the core code the showcase tests are exercising, observe coverage drop/rise as we continue developing and also help us determine if it is safe to replace the existing handwritten integration tests. Given this purpose, do we think having a combined test coverage of `api-common`, `gax-java` and `showcase` tells a more complete story?

cc/ @burkedavison

Reply:
Makes sense to me. If it's part of the 'platform runtime', let's include it. In the future this may also include java-core if we have a handwritten layer on top of showcase.

Reply:
Sounds good! Will look into adding this in as well.

Reply:
Done, updated the README and PR description as well.